Sources/TestingInternals/include/Includes.h

```diff
@@ -75,10 +75,19 @@
 #include <wasi/libc-environ.h>
 #endif
 
+#if !SWT_NO_CURL
```

Comment:
Should this be under `!defined(_WIN32)`?

Reply:
Nope, because Windows doesn't have this header anyway.

Reply:
I'm not sure I understand this. The header is a curl header, and this header has a `_WIN32` path below.

Reply:
If your Windows SDK happens to have the libcurl headers, we will end up including this header, but we just won't do anything with them. The same can be said of the other headers here. We added a `SWT_NO_CURL` check because not all Linux/POSIX-like variants will have the libcurl headers and some custom configuration may be needed.